### ISSUE TITLE
MissingJobPermissions false negative: reusable workflow call

### ISSUE THREAD

**Issue by TWiStErRob** [TWiStErRob/net.twisterrob.ghlint#127]
### Version

0.3.1

### Reproduction steps

1. File `.github/workflows/x.yml` with the following content:
   ```yaml
   on: push
   jobs:

     validate:
       name: "🦺 Validation"
       uses: TWiStErRob/github-workflows/.github/workflows/validate.yml@94f2ec4f96e146200f801fe7e09310c79cb2d2b3 # v2
   ```

2. Run `ghlint` with the following arguments:
   ```shell
   ghlint --console .github/workflows/x.yml
   ```

### Expected behavior

MissingJobPermissions

### Actual behavior

\-
